it/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/CaptureBusinessPostcodeControllerISpec.scala

```diff
@@ -48,7 +52,23 @@ class CaptureBusinessPostcodeControllerISpec extends ComponentSpecHelper with Ca
       )
     }
 
+      "when the user has submitted an empty form, the page" should {
+        lazy val authStub = stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
+        lazy val result = post(s"/$testJourneyId/business-postcode")(
+          "business_postcode" -> "111 111"
+        )
+        testCaptureVatRegistrationPostcodeRedirectViewTests(result, authStub)
+
+        "return a BAD_REQUEST" in {
+          stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
+          lazy val result = post(s"/$testJourneyId/vat-registration-date")()
+
+          result.status mustBe BAD_REQUEST
+        }
+      }
+
     "return a BAD_REQUEST if the postcode is missing" in {
+      stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
```

Comment:
This and the test above it are the same test. You don't need this one (lines 70-77). But in the test above it says that the user is submitting an empty form but you're supplying a postcode in the post. 
The business postcode needs to be blank (see below comment) and then the view tests will need to be the one that tests for missing postcode (testCaptureBusinessPostcodeMissingErrorViewTests) - explained further below.